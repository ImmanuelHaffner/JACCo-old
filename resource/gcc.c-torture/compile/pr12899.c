

void
bb_getopt_ulflags (char *s)
{
  for (;;)
    if (s[1])
      do
 s++;
      while (*s);
}
