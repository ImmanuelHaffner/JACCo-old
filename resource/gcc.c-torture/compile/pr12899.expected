0
pr12899.c:3:1 keyword void
pr12899.c:4:1 identifier bb_getopt_ulflags
pr12899.c:4:19 punctuator (
pr12899.c:4:20 keyword char
pr12899.c:4:25 punctuator *
pr12899.c:4:26 identifier s
pr12899.c:4:27 punctuator )
pr12899.c:5:1 punctuator {
pr12899.c:6:3 keyword for
pr12899.c:6:7 punctuator (
pr12899.c:6:8 punctuator ;
pr12899.c:6:9 punctuator ;
pr12899.c:6:10 punctuator )
pr12899.c:7:5 keyword if
pr12899.c:7:8 punctuator (
pr12899.c:7:9 identifier s
pr12899.c:7:10 punctuator [
pr12899.c:7:11 constant 1
pr12899.c:7:12 punctuator ]
pr12899.c:7:13 punctuator )
pr12899.c:8:7 keyword do
pr12899.c:9:2 identifier s
pr12899.c:9:3 punctuator ++
pr12899.c:9:5 punctuator ;
pr12899.c:10:7 keyword while
pr12899.c:10:13 punctuator (
pr12899.c:10:14 punctuator *
pr12899.c:10:15 identifier s
pr12899.c:10:16 punctuator )
pr12899.c:10:17 punctuator ;
pr12899.c:11:1 punctuator }
